tomviz/ImageStackDialog.cxx

```diff
@@ -0,0 +1,39 @@
+/******************************************************************************
+
+  This source file is part of the tomviz project.
+
+  Copyright Kitware, Inc.
+
+  This source code is released under the New BSD License, (the "License").
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+******************************************************************************/
+
+#include "ImageStackDialog.h"
+#include "ui_ImageStackDialog.h"
+
+#include "ImageStackModel.h"
+
+#include <QDialog>
+
+namespace tomviz {
+
+ImageStackDialog::ImageStackDialog(QWidget* parent, ImageStackModel* tableModel)
+  : QDialog(parent), m_ui(new Ui::ImageStackDialog)
+{
+  m_ui->setupUi(this);
+  m_ui->tableView->setModel(tableModel);
+  m_ui->tableView->resizeColumnsToContents();
+  m_ui->tableView->horizontalHeader()->setSectionResizeMode(
+    0, QHeaderView::Stretch);
+}
+
+ImageStackDialog::~ImageStackDialog()
```

Comment:
Minor, but avoid empty destructors - you need a non-inline dtor, but can have the compile generate one using `ImageStackDialog::~ImageStackDialog() = default;` which also has the advantage of letting the compiler know you are not doing anything fancy in the dtor so it may optimize.